src/cdk-experimental/menu/menu-group.ts

```diff
@@ -19,18 +30,37 @@ import {MenuItem} from './menu-item-interface';
   host: {
     'role': 'group',
   },
+  providers: [{provide: UniqueSelectionDispatcher, useClass: UniqueSelectionDispatcher}],
 })
-export class CdkMenuGroup {
+export class CdkMenuGroup implements AfterContentInit, OnDestroy {
   /** Emits the element when checkbox or radiobutton state changed  */
-  @Output() change: EventEmitter<MenuItem> = new EventEmitter();
-
-  /**
-   * Emits events for the clicked MenuItem
-   * @param menuItem The clicked MenuItem to handle
-   */
-  _registerTriggeredItem(menuItem: MenuItem) {
-    if (menuItem.role !== 'menuitem') {
-      this.change.emit(menuItem);
-    }
+  @Output() change: EventEmitter<CdkMenuItem> = new EventEmitter();
+
+  /** List of menuitemcheckbox or menuitemradio elements which reside in this group */
+  @ContentChildren(CdkMenuItemSelectable, {descendants: true})
+  private readonly _selectableItems: QueryList<CdkMenuItemSelectable>;
+
+  /** Emitter used to unsubscribe from selectable item click events */
+  private readonly _nextChanges: EventEmitter<void> = new EventEmitter();
```

Comment:
I'm used to seeing this named 'destroy'. (Also, it's usually a ReplaySubject, so it can be reused for multiple subscriptions.)